report-jtreg/src/main/resources/io/jenkins/plugins/report/jtreg/JenkinsReportJckGlobalConfig/config.jelly

```diff
@@ -78,4 +78,35 @@
             <f:repeatableDeleteButton/>
         </f:repeatable>
     </f:section>
+
+    <f:section title="Failed Test Links">
+        <div>
+            <p>
+                This section is used for generating automatic links which will be shown next to all failed tests on the jtreg report page.
+                It uses the same macro system as described in the <i>Comparator Links</i> section, however, these links can lead to any tool, not just the comparator.
+            </p>
```

Comment:
This <p> may be duplicatin the info about %{TESTNAME}, wdyt? Or at least mention it exists...